package.json

```diff
@@ -70,12 +73,10 @@
     "suitcss-utils-position": "^1.0.1",
     "suitcss-utils-size": "^2.0.0",
     "suitcss-utils-text": "^1.0.0",
-    "suitcss-utils": "^1.0.0",
     "umbrellajs": "^2.6.7"
   },
   "scripts": {
-    "prestart": "npm install",
```

Comment:
Apparently my reply to this was lost when GitHub was down yesterday morning. Sorry about that!

As a bit of history, historically our designers have interacted with this more than our developers. We encountered a lot of headaches when dependencies were added or changed, which required us to constantly nag each other to "remember to run `npm install`." Inevitably someone would forget, we'd see issues or errors, etc. The team was then running `npm install` every time _anyway_ just in case, so it made sense to add this `prestart` command to save everyone a step.

This was about six major Node versions ago, so it could be that there's a better way to manage that today. But I'd rather replace the solution (ideally as a separate effort from this PR) than remove it entirely.

Reply:
OK, I will change it back